src/lib.rs

```diff
@@ -225,9 +227,28 @@ impl Command {
     }
 }
 
+/// Type parameter: SHTC1.
+pub struct ShtC1;
+/// Type parameter: SHTC3.
+pub struct ShtC3;
```

Comment:
You could also add SHTW2 and SHTW2_alt.

Reply:
If the implementation for the SHTC3 and the SHTW2 are identical, we should probably not create multiple variants, right?

Reply:
Covered by `ShtBasic` and a `shtw2` factory function.